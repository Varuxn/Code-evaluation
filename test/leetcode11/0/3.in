10 23
5 6 77
2 7 737
9 2 550
1 5 469
1 3 744
4 5 837
2 8 533
0 2 752
0 4 498
0 7 747
7 6 94
9 0 921
8 1 733
3 5 707
0 8 479
2 4 39
5 7 370
0 5 442
0 3 699
9 8 230
2 6 10
0 1 686
3 9 979
