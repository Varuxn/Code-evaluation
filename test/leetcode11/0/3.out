0 3 10 14 15 17 18 19 20 

